### ISSUE TITLE
Synthetics via GitHub Action: unable to trigger tests

### ISSUE THREAD

**Issue by erodewald** [DataDog/datadog-ci#480]
**Describe what happened**
I am using the GitHub Action which seems to leverage `datadog-ci`/`synthetics` to trigger my Synthetic tests. I had it working successfully, and yesterday the action began to fail, with the output `ERROR: unable to trigger tests`. 

**Steps to reproduce the issue:**
Action configuration: 
```yaml
      - name: Run Datadog Synthetics tests
        uses: DataDog/synthetics-ci-github-action@v0.2.2
        with:
          api_key: ${{secrets.DD_API_KEY}}
          app_key: ${{secrets.DD_APP_KEY}}
```
Global config:  (values are `sed` replaced on execution)
```json
{
    "global": {
        "headers": {
            "authorization": "%%SIGNATURE%%",
            "date": "%%DATE%%",
            "host": "%%HOST%%"
        }
    }
}
```
Test config prior to job execution:
```json
{
	"tests": [{
		"id": "5wj-rcg-7b4",
		"config": {
			"headers": {
				"authorization": "%%SIGNATURE%%",
				"date": "%%DATE%%",
				"host": "%%HOST%%"
			}
		}
	}]
}
```

Test config after replacement: 
```json
{
	"tests": [{
		"id": "5wj-rcg-7b4",
		"config": {
			"headers": {
				"authorization": "Signature keyId=\"***\",created=1645007819,algorithm=\"hs2019\",headers=\"(request-target) host date\",signature=\"SNIP=\"",
				"date": "Wed, 16 Feb 2022 10:36:59 GMT",
				"host": "enterprised.SNIP.com"
			}
		}
	}]
}
```

**Expected behaviour:**
From a working job Feb 16 2022 4:37 AM CST: 
```
Finding files in /home/ubuntu/actions-runner/_work/dax-synthetics/dax-synthetics/{,!(node_modules)/**/}*.synthetics.json

Got test files:
  - enterprise.synthetics.json

[5wj-rcg-7b4] Trigger test "[CI-only] Enterprise API Assume Role" (non-blocking)
[5wj-rcg-7b4] Waiting results for "[CI-only] Enterprise API Assume Role"


=== REPORT ===
Took 8618ms
```

**Actual behaviour:**
```
Finding files in /home/ubuntu/actions-runner/_work/dax-synthetics/dax-synthetics/{,!(node_modules)/**/}*.synthetics.json

Got test files:
  - enterprise.synthetics.json

[5wj-rcg-7b4] Trigger test "[CI-only] Enterprise API Assume Role"
[5wj-rcg-7b4] Waiting results for "[CI-only] Enterprise API Assume Role"

 ERROR: unable to trigger tests
```

**Additional context**
- my organization has a Business account with Datadog
 - OS version - Typical github action Linux x64 runners
 - Node version - its the Action, which I assume makes this a known quantity. It has no GitHub so I'm posting here. 
 - I did some digging and found that: 
   - the error string originated [here](https://github.com/DataDog/datadog-ci/blob/0bfda364ac7de6b35023fbd09e8849193187b683/src/commands/synthetics/run-test.ts#L91-L97)
   - because it [was a 5xx](https://github.com/DataDog/datadog-ci/blob/0bfda364ac7de6b35023fbd09e8849193187b683/src/commands/synthetics/api.ts#L121) (I cannot see any further details and I have all known debug flags on – is there more I can turn on in the Action's config?)

**Comment by peculiarism** [comment]
Hi @erodewald, thank you for raising this. We will get on this ASAP.

**Comment by peculiarism** [comment]
@erodewald we have narrowed down the impacted versions to `v0.2.1` and `v.0.2.2`. We will have a fix in place soon, but to ensure your Synthetic tests continue to run, we would encourage you to use `v.0.2.0` in the meantime. 

I will update this issue when a fix is released, thank you for your patience!

**Comment by paulvidal** [comment]
@peculiarism a small note also, I don't know why but 

```
 ERROR: unable to trigger tests
```

even with all the failure flags like `--failOnTimeout --failOnCriticalErrors` does not return an error code it seems, and thus appear as an ok step in my CI (we took some time to discover the bug because we thought the CI was green, but tests were just not executed)

**Comment by paulvidal** [comment]
also, what do you mean by using version `v0.2.1` ? the latest version I see is `v0.17.12` :) Are we talking about the same lib ?

**Comment by peculiarism** [comment]
Hi @paulvidal, apologies I should have been clearer, for the GitHub Action related to this issue `v0.2.0` is a stable release. 

We have released `v.0.17.13` for `datadog-ci`, could you update to that version and let me know if you are still seeing the above error?

**Comment by paulvidal** [comment]
Hi @peculiarism :) I had to revert to `0.17.8`, it was not working on `0.17.13` :)

Also, do you think you could fix this part that does not throw an error, I think it is important to return an error exit code 🙇‍♂️ Many thanks 

```
ERROR: unable to trigger tests
```

**Comment by paulvidal** [comment]
all this is fixed in versions 0.18.x :) I successfully upgraded thank you 🙏
